src/api/api.js

```diff
@@ -0,0 +1,17 @@
+const getRestaurants = async () => {
+  const response = await fetch('http://localhost:3000/restaurants');
+  const data = await response.json();
+  return data;
+};
+
+const addNewRestaurant = async (newRestaurant) => {
+  await fetch('http://localhost:3000/restaurants', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(newRestaurant),
+  });
+};
+
+export { getRestaurants, addNewRestaurant };
```

Comment:
```suggestion
export const getRestaurants = async () => {
  const response = await fetch('http://localhost:3000/restaurants');
  const data = await response.json();
  return data;
};

export const addNewRestaurant = async (newRestaurant) => {
  await fetch('http://localhost:3000/restaurants', {
    method: 'POST',
    headers: {
      'Content-Type': 'application/json',
    },
    body: JSON.stringify(newRestaurant),
  });
};
```
이렇게 작성하는게 작업하기도 더 편하실 것 같아요. 가독성에도 좋구요.

Reply:
이 부분 반영하여 리팩터링 진행하겠습니다.🛠️